lib/ansible/module_utils/facts/system/lsmod.py

```diff
@@ -0,0 +1,57 @@
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+import os
+from collections import OrderedDict
```

Comment:
`OrderedDict` is not available in Python 2.6, which is why CI tests are failing.

Reply:
Ah, it doesn't work anyway, so I just rescoped and amended the commit. Thought you guys had already phased out py2.6 support.

Reply:
Python 2.6 is no longer supported on the controller. Modules (and therefore module_utils) still support Python 2.6.